src/main/java/com/algorand/sdkutils/generators/Utils.java

```diff
@@ -0,0 +1,138 @@
+package com.algorand.sdkutils.generators;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.StringTokenizer;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class Utils {
+	
+	public static DiffResult showDifferentces(
```

Comment:
You can use assertJ instead of this `assertThat(jsonA).isEqualTo(jsonB)` will tell you what part of the line is different if they aren't equal.